moped-editor/src/components/GridTable/Filters.js

```diff
@@ -1,6 +1,5 @@
-import React, { useCallback, useEffect, useState, useMemo } from "react";
+import React, { useCallback, useState, useMemo } from "react";
 import { useSearchParams } from "react-router-dom";
-import { v4 as uuidv4 } from "uuid";
```

Comment:
Since the shape of `filterParameters` is now an array, we can use the index of a filter in the array to point to the filter to edit, delete, etc. Previously, we used an object so we needed to assign a unique ID so that we could point to the right filter in the object.